libafl/src/feedbacks/map.rs

```diff
@@ -132,6 +216,24 @@ fn saturating_next_power_of_two<T: PrimInt>(n: T) -> T {
     }
 }
 
+/// A filter that only saves values which are at least the next pow2 class
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct MapAflFilter {}
+impl<T> MapFindFilter<T> for MapAflFilter
+where
+    T: PrimInt + Default + Copy + 'static + serde::Serialize + serde::de::DeserializeOwned,
+{
+    #[inline]
+    fn is_interesting(old: T, new: T) -> bool {
+        // check if the bit is already set
+        if new & old == new {
```

Comment:
I gues you could just write `(new & old) != new` instead of having the explicit if

Reply:
oh, much better!